neoload/README.md

```diff
@@ -0,0 +1,49 @@
+# NeoLoad integration
+
+## Overview
+
+[Tricentis NeoLoad][1] simplifies and scales performance testing for APIs and microservices, as well as end-to-end application testing through protocol and browser-based capabilities.
+
+With the NeoLoad integration, you can track performance metrics of NeoLoad tests to:
+
+- Correlate application performance with load testing metrics.
+- Analyze and visualize NeoLoad metrics using the NeoLoad Datadog Dashboard or Metrics Explorer.
+
+## Setup
+
+### Configuration
+
+For detailed instructions on NeoLoad configuration, follow the [NeoLoad documentation][2]. Choose which metrics to send in the "Push Counters" configuration of the Datadog Connector in NeoLoad. Available since NeoLoad 9.1.
+
+When Datadog receives `NeoLoad.*` metrics, the NeoLoad integration tile is installed automatically, and the default NeoLoad dashboard is added to your dashboard list.
+
+![NeoLoad Dashboard][7]
+
+## Data Collected
+
+### Metrics
+
+See [metadata.csv][3] for a list of metrics provided by this integration.
+
+### Service Checks
+
+NeoLoad does not include any service checks.
+
+### Events
+
+All NeoLoad performance tests events are sent to your [Datadog Events Explorer][4].
+NeoLoad sends events to the Datadog API when a performance test starts and ends.
+Set the option in the "Push Counters" configuration of the Datadog Connector in NeoLoad. Available since NeoLoad 9.1.
+
+## Troubleshooting
+
+Need help? Contact [Datadog support][5] or [Tricentis NeoLoad support][6].
+
+[1]: https://www.tricentis.com/products/performance-testing-neoload
+[2]: https://documentation.tricentis.com/neoload/9.1/en/content/reference_guide/datadog.htm
```

Comment:
This link redirects to another [link](https://documentation.tricentis.com/neoload/9.1/en/content/reference_guide/datadog.htm )

Can you update the existing link with the one above?